astron/config/cluster_yaml.yml

```diff
@@ -81,7 +81,7 @@ roles:
         max: 399999999
       backend:
         type: yaml
-        directory: ../databases/astrondb
+        foldername: ../databases/astrondb
```

Comment:
Why would you change this? i believe recent versions of astron use "directory" over foldername. This could break it for some people who are using a more recent version of astron.

Reply:
i forgot to set gitignore but the reason is astron is not very supportive of mac and most recent one on mac uses foldername

Reply:
Please revert this change.

Reply:
already did